etc/config/hooks/set-inputrc-completion.chroot

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# Description: Enable case-insensitive tab completion in /etc/inputrc
+
+echo "P: Begin executing set-inputrc-completion chroot hook..."
```

Comment:
I think this is now taken care of in https://github.com/elementary/default-settings/blob/e9c449aad3913f8dd7dafab1a20c067f2414ed59/.inputrc

So we might not need this script